client/js/App.js

```diff
@@ -1,12 +1,14 @@
 import React, { Component } from "react";
-import { Text, View } from "react-native";
+import LoginContainer from "./screens/Login";
+import client from "../config/api";
```

Comment:
Since you changed your project structure it's throwing errors now. Please make sure all of your imports exports are correct.